node/src/command.rs

```diff
@@ -49,7 +49,8 @@ impl SubstrateCli for Cli {
     fn load_spec(&self, id: &str) -> Result<Box<dyn sc_service::ChainSpec>, String> {
         Ok(match id {
             "dev" => Box::new(chain_spec::development_config()?),
-            "" | "vitalam" => Box::new(chain_spec::vitalam_config()?),
+            "" | "inteli-stage" => Box::new(chain_spec::inteli_stage_config()?),
+            "vitalam" => Box::new(chain_spec::vitalam_config()?),
```

Comment:
so our network runs all the chsins at once? or specify a single one?

Reply:
specify one, that's in a `match` statement